### PR TITLE
Feature/cp 3408/styled components

### PR DESCRIPTION
**Pull request by nf2000**
Styling components

https://chillibean.atlassian.net/browse/CP-3408

### CODE REVIEW

File: nims-notes-things/src/App.js [hunk 1]
```diff
@@ -1,9 +1,14 @@
+import React from 'react';
+// import pencil from './penLogo.png';
 import "./App.css";
 import Form from "./Form";
 
 function App() {
   return (
     <div className="App">
+      <div className="App-header">
```
Comment (damonspencerroberts) [inline nims-notes-things/src/App.js]: create a styled component for this

File: nims-notes-things/src/App.js [hunk 2]
```diff
@@ -1,9 +1,14 @@
+import React from 'react';
+// import pencil from './penLogo.png';
 import "./App.css";
 import Form from "./Form";
 
 function App() {
   return (
     <div className="App">
```
Comment (damonspencerroberts) [inline nims-notes-things/src/App.js:8]: create a container component or styled component for this with App styles

Comment (damonspencerroberts) [inline nims-notes-things/src/App.js:8]: You can also wrap you app in a theme https://styled-components.com/docs/advanced

File: nims-notes-things/src/App.js [hunk 3]
```diff
@@ -1,9 +1,14 @@
...
+      <div className="App-header">
+      <h1> Nims Notes Things </h1>
```
Comment (damonspencerroberts) [inline nims-notes-things/src/App.js]: Indentation

File: nims-notes-things/src/Form.js [hunk 4]
```diff
@@ -23,56 +24,77 @@ const Form = (props) => {
...
+        <InnerForm>
+          {empty.name && <ErrorMessage>Name field is required</ErrorMessage>}
+          <Input
+            name="name"
+            placeholder="Please enter Your name"
+            onChange={(e) => handleOnChange(e, "name")}
+            onBlur={(e) => handleOnBlur(e.target.value, "name")}
+            value={formData.name}
+          />
+          <ErrorMessage>
```
Comment (damonspencerroberts) [inline nims-notes-things/src/Form.js]: no need to wrap the error message here...
`{empty.date && <ErrorMessage>Date field is required</ErrorMessage>}` is enough

File: nims-notes-things/src/Form.js [hunk 5]
```diff
@@ -23,56 +24,77 @@ const Form = (props) => {
...
-      <h5>Date: {formData.date}</h5>
+          <Input
+            name="date"
+            type="date"
+            placeholder="Please enter the date"
+            onChange={(e) => handleOnChange(e, "date")}
+            onBlur={(e) => handleOnBlur(e.target.value, "date")}
+            value={formData.date}
+          />
+          <ErrorMessage>
```
Comment (damonspencerroberts) [inline nims-notes-things/src/Form.js]: remove wrapper

**Review by damonspencerroberts — CHANGES_REQUESTED** [review]
Couple of small things, great job

File: nims-notes-things/src/App.js (hunk 2 above)
Comment (nf2000) [reply nims-notes-things/src/App.js:8]: I didn't need that image imported there so i've taken it out

**Pull request closed without merging**